components/centraldashboard/public/components/dashboard-view.css

```
@@ -52,25 +75,11 @@ paper-card.thin {
     grid-column: 3
 }
 
-#Quick-Links .link {
-    width: 80%;
-    margin: .5em auto;
-    border: 1px solid #eeeeef;
-    padding: .5em 1em;
-    border-radius: 5px;
-    @apply --layout-horizontal;
-}
-
-#Quick-Links .link.more-coming {
-    opacity: .4;
-    font-style: italic;
-    pointer-events: none
-}
-
-#Quick-Links .link .button {
-    color: var(--accent-color);
-    background: rgba(0, 125, 252, 0.25);
-    border-radius: 50%
+#Platform-Links header {
+    font-size: 18px;
```

```suggestion
    font-size: 1.25em;
```

Try to use `em`s since they scale according to ancestors and is easier to build responsive pages.

_**Note:** `1.25em` may not be the best, please experiment and find out what size is best_

---

1.25 seems to come out pretty well.